frontend/src/App.tsx

```diff
@@ -45,14 +45,14 @@ const App = () => {
           <Route exact path='/billing' component={Billing} />
           <Route exact path='/checkout/:id' component={Checkout} />
           <Route exact path='/guests' component={Guests} />
-          <Route exact path='/guests/:id' component={GuestDetail} />
           <Route exact path='/guests/new' component={NewGuest} />
+          <Route exact path='/guests/:id' component={GuestDetail} />
```

Comment:
Just out curiosity, why this change?

Reply:
Sorry I missed this comment. I go on detail in the second paragraph of the PR description, but long story short, if `/guests/:id` goes first, the path `/guests/new` will trip `/guests/:id`, and React Router will interpret it it as `/guests` with an ID of `new`. This leads to the wrong component rendering.